eras/conway/impl/CHANGELOG.md

```diff
@@ -17,11 +17,23 @@
   `gasStakePoolVotes` in `GovernanceActionState`
 * Removal of `reRoles` from `RatifyEnv` as no longer needed
 * Addtion of `reStakePoolDistr` to `RatifyEnv`
-* Remove `VoterDoesNotHaveRole` as no longer needed from `ConwayTallyPredFailure`
+* Remove `VoterDoesNotHaveRole` as no longer needed from `ConwayGovPredFailure`
 * Added `ConwayEpochPredFailure`
 * Added instance for `Embed (ConwayRATIFY era) (ConwayEPOCH era)`
 * Removed instance for `Embed (ConwayRATIFY era) (ConwayNEWEPOCH era)`
 * Changed superclasses of `STS (ConwayEPOCH era)` and `STS (ConwayNEWEPOCH era)`
+* Rename:
```

Comment:
These need to go into a new `## 1.7.0.0` section and cabal version needs a bump too, since `1.6.0.0` has already been released